<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContextSettingsPage</class>
 <widget class="QWidget" name="ContextSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Context</string>
  </property>
  <layout class="QVBoxLayout" name="layout_contextsettingspage">
   <item>
    <widget class="QGroupBox" name="context_custom_text_group">
     <property name="title">
      <string>Custom text settings</string>
     </property>
     <layout class="QVBoxLayout" name="layout_context_custom_text_group">
      <item>
       <widget class="QFrame" name="frame_custom_context1">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame1">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <layout class="QGridLayout" name="gridLayout1">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="1">
          <widget class="QLineEdit" name="context_custom_text1">
           <property name="text">
            <string>%title - %artist%</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QToolButton" name="context_exp_chooser1">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_title">
           <property name="text">
            <string>Title</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_summary">
           <property name="text">
            <string>Summary</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="context_custom_text2">
           <property name="text">
            <string>%album%</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QToolButton" name="context_exp_chooser2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="context_order_group">
     <property name="title">
      <string>Enable Items</string>
     </property>
     <layout class="QVBoxLayout" name="layout_context_order_group">
      <item>
       <widget class="QFrame" name="frame_custom_context2">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_17">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="context_item1_enable">
        <property name="text">
         <string>Album</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="context_item2_enable">
        <property name="text">
         <string>Technical Data</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="context_item3_enable">
        <property name="text">
         <string>Engine and Device</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="context_item4_enable">
        <property name="text">
         <string>Albums by Artist</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="context_item5_enable">
        <property name="text">
         <string>Song Lyrics</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="spacer_bottom">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>32</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <action name="action_artist">
   <property name="text">
    <string notr="true">%artist%</string>
   </property>
   <property name="toolTip">
    <string>Add song artist tag</string>
   </property>
  </action>
  <action name="action_album">
   <property name="text">
    <string notr="true">%album%</string>
   </property>
   <property name="toolTip">
    <string>Add song album tag</string>
   </property>
  </action>
  <action name="action_title">
   <property name="text">
    <string notr="true">%title%</string>
   </property>
   <property name="toolTip">
    <string>Add song title tag</string>
   </property>
  </action>
  <action name="action_albumartist">
   <property name="text">
    <string notr="true">%albumartist%</string>
   </property>
   <property name="toolTip">
    <string>Add song albumartist tag</string>
   </property>
  </action>
  <action name="action_year">
   <property name="text">
    <string notr="true">%year%</string>
   </property>
   <property name="toolTip">
    <string>Add song year tag</string>
   </property>
  </action>
  <action name="action_composer">
   <property name="text">
    <string notr="true">%composer%</string>
   </property>
   <property name="toolTip">
    <string>Add song composer tag</string>
   </property>
  </action>
  <action name="action_performer">
   <property name="text">
    <string notr="true">%performer%</string>
   </property>
   <property name="toolTip">
    <string>Add song performer tag</string>
   </property>
  </action>
  <action name="action_grouping">
   <property name="text">
    <string notr="true">%grouping%</string>
   </property>
   <property name="toolTip">
    <string>Add song grouping tag</string>
   </property>
  </action>
  <action name="action_disc">
   <property name="text">
    <string notr="true">%disc%</string>
   </property>
   <property name="toolTip">
    <string>Add song disc tag</string>
   </property>
  </action>
  <action name="action_track">
   <property name="text">
    <string notr="true">%track%</string>
   </property>
   <property name="toolTip">
    <string>Add song track tag</string>
   </property>
  </action>
  <action name="action_genre">
   <property name="text">
    <string notr="true">%genre%</string>
   </property>
   <property name="toolTip">
    <string>Add song genre tag</string>
   </property>
  </action>
  <action name="action_length">
   <property name="text">
    <string notr="true">%length%</string>
   </property>
   <property name="toolTip">
    <string>Add song length tag</string>
   </property>
  </action>
  <action name="action_playcount">
   <property name="text">
    <string notr="true">%playcount%</string>
   </property>
   <property name="toolTip">
    <string>Add song play count</string>
   </property>
  </action>
  <action name="action_skipcount">
   <property name="text">
    <string notr="true">%skipcount%</string>
   </property>
   <property name="toolTip">
    <string>Add song skip count</string>
   </property>
  </action>
  <action name="action_newline">
   <property name="text">
    <string notr="true">%newline%</string>
   </property>
   <property name="toolTip">
    <string>Add a new line if supported by the notification type</string>
   </property>
  </action>
  <action name="action_filename">
   <property name="text">
    <string>%filename%</string>
   </property>
   <property name="toolTip">
    <string>Add song filename</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
